jabgui/src/main/java/org/jabref/gui/preview/assets/PreviewText.java

```diff
@@ -0,0 +1,20 @@
+package org.jabref.gui.preview.assets;
+
+public final class PreviewText {
+
+    private PreviewText() {
+    }
+
+    public static String getPreviewText(String baseUrl, String text) {
+        return """
+                <html>
+                    <head>
+                        <base href="%s">
+                    </head>
+                    <body id="previewBody">
+                        <div id="content"> %s </div>
+                    </body>
+                </html>
+                """.formatted(baseUrl, text);
+    }
+}
```

Comment:
Please undo this. This is weird.

Reply:
You can keep it as a constant, but put it in PreviewViewer. We dont need it in any other place.